docs/backplane/requirements/acs/10-acs-managed-service.md

```diff
@@ -32,5 +63,49 @@ Permissions at cluster scope.
 * get pods and pod logs
 * view deployments
 * get deployments
+* view routes
+* view egress firewalls
+* view roles and rolebindings
+* patch persistentvolumeclaims
```

Comment:
Missing get/list/watch for PVC for backplane-acs-admins-project Role?

Reply:
present in cluster role